src/mcts/search.h

```diff
@@ -193,7 +193,9 @@ class Search {
 class SearchWorker {
  public:
   SearchWorker(Search* search, const SearchParams& params)
-      : search_(search), history_(search_->played_history_), params_(params) {}
+      : search_(search), history_(search_->played_history_), params_(params) {
+    minibatch_.reserve(32);
```

Comment:
This also doesn't help in my tests

Reply:
I output the actual size the minibatch grows to, which usually went above 250, and changed this to a much larger number in updated code (300). After making this change the performance different, for me, was significant.

Reply:
This is probably compiler/system dependent (still doesn't help here).